src/Orchard/Orchard.Framework.csproj

```diff
@@ -108,9 +108,7 @@
       <RequiredTargetFramework>3.5</RequiredTargetFramework>
     </Reference>
     <Reference Include="System.Data" />
-    <Reference Include="System.Net.Http">
-      <Private>True</Private>
-    </Reference>
```

Comment:
Not sure why, but setting this reference private (copy local) with the version number pinned to 4.2.0.0 causes the same error when running the "normal" application as the spec app had (not being able to resolve the WorkContext).